core/src/main/resources/messages/messages.properties

```diff
@@ -28,14 +28,13 @@ notification.message.body=Tap to open message
 error.invalid.request=Invalid request
 error.invalid.uuid=String ''{0}'' is not a valid UUID
 error.invalid.date=String ''{0}'' is not a valid date
-error.invalid.date.too.early=Date ''{0}}'' is too early. Earliest possible date is ``{1}}``
+error.invalid.date.too.early=Date ''{0}'' is too early. Earliest possible date is ``{1}``
```

Comment:
Good catch... Another one: I see two ticks `''` vs. two back-ticks ````, which we may correct too

Reply:
There were a few more of these. Replaced them all